src/main/java/ch/njol/skript/SkriptConfig.java

```diff
@@ -79,21 +61,21 @@ public class SkriptConfig {
 				}
 			});
 	
-	static final Option<Boolean> checkForNewVersion = new Option<>("check for new version", false)
+	public static final Option<Boolean> checkForNewVersion = new Option<>("check for new version", false)
```

Comment:
I don't think it really matters whether these fields are public, but it also doesn't matter where this class is. If you'd prefer to keep their access the same, then there would be no issue to move this class.

Reply:
I think I'd prefer new stuff to stay in the skriptlang package if possible so I'll keep them public and just edit the formatting like you suggested, lime.

Reply:
well actually looking at it the whole file needs to be updated in terms of formatting, they're all camelCase and PSF

Reply:
> I don't think it really matters whether these fields are public, but it also doesn't matter where this class is. If you'd prefer to keep their access the same, then there would be no issue to move this class.

The file location does matter as it's keeping the file within the `ch.njol.skript` package to allow the fields to keep their package access.

> I think I'd prefer new stuff to stay in the skriptlang package if possible so I'll keep them public and just edit the formatting like you suggested, lime.

Yes, in the case you don't require a class existing from the old packages. This is a new class that requires access to the old class, and will be renamed in the future when the whole `ch.njol.skript` package gets renamed.

Reply:
I think the best route here would be to fix up SkriptConfig in a separate PR (which I will do shortly) so that it doesn't rely on public static fields. This way the metrics class just has to call something like `Skript.getConfig().getXYZValue()` and there's no package concerns. So for now I'm going to leave it as is, with the knowledge that it will be replaced soon.